phabricatoremails/source.py

```diff
@@ -5,10 +5,10 @@
 import json
 import pathlib
 from dataclasses import dataclass
-from typing import Protocol, Dict
+from typing import Optional, Protocol, Dict
```

Comment:
Off topic: we should get our usual combination of `ruff`/`black` running on this repo at some point, for import sorting etc. :)

Reply:
It's got `black` running in the tests https://github.com/mozilla-conduit/phabricator-emails/blob/master/tests/test_style.py#L16-L17

This code base hasn't been touched in a while (: